Makefile.toml

```diff
@@ -0,0 +1,327 @@
+env_scripts = [
+'''
+#!@duckscript
+crates = get_env CRATES
+crates = array %{crates}
+NAMES = array
+CRATE_NAMES = array
+YAMLS = array
+PATCHED_SVDS = array
+FORMATTED_SVDS = array
+RUST_DIRS = array
+for crate in ${crates}
+    crate_yamls = glob_array devices/${crate}*.yaml
+    for yaml in ${crate_yamls}
+        serie = substring ${yaml} 8
+        serie = substring ${serie} -5
+        array_push ${NAMES} ${serie}
```

Comment:
`NAMES = concat ${NAMES} " " ${serie}`

and drop the array_join below?

Reply:
Can I avoid using substring here somehow? Take `serie` directly from glob request?

Reply:
Don't know about that.